tests/e2e/specs/BoundaryErrorEvent.spec.js

```diff
@@ -1,36 +1,97 @@
 import {
   dragFromSourceToDest,
   getElementAtPosition,
-  removeIndentationAndLinebreaks,
+  getComponentsEmbeddedInShape,
+  getPositionInPaperCoords,
+  waitToRenderAllShapes,
 } from '../support/utils';
 import { nodeTypes } from '../support/constants';
 
-describe.only('Boundary Error Event', () => {
+describe('Boundary Error Event', () => {
   beforeEach(() => {
```

Comment:
I think this `beforeEach()` call is unnecessary with the other changes that we've introduced recently. Can you remove it?